style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+@media only screen and (max-width: 768px) {
+  .container {
+    width: 100%;
+    height: 100%;
+    background-image: url("./pictures/header-shapes\ mobile.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    padding-left: 24px;
+    padding-right: 24px;
+    padding-top: 50px;
+  }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    height: 20%;
+    align-items: center;
+  }
+
+  header h3 {
+    color: #6070ff;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  .icon {
+    color: #6070ff;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  ul {
+    width: 50%;
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+  }
+
+  p {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  h1 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+  }
+
+  h5 {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .socialMedia {
+    font-size: 22px;
+    color: #505f79;
+    font-weight: 500;
+  }
+}
```

Comment:
- [ ] As stated the project requirement Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust the overlapping between the hamburger menu and background image .

The Figma design Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/219175315-00e68f86-de29-4460-b611-82cba165fbd5.PNG)

Your Project Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/219175506-dfaa2f26-bf18-4ece-8e3d-dac8fce85764.PNG)

Reply:
@Abenezer-Tilahun i made all the changes can you please give me more advices